### DISCUSSION TITLE
yi-hack-MStar 0.4.9 chinese version YI Dome still have to Use proxychains ng to connect your app?

### DISCUSSION THREAD

**Discussion by tummerry** [roleoroleo/yi-hack-MStar#488] [Q&A]
yi-hack-MStar 0.4.9 chinese version YI Dome still have to Use proxychains ng to connect your app?
Is there a way that I don't have to use it?
Reconnected and not connected  must do proxychains ng new

**Comment by roleoroleo** [comment]
What do you mean with "your app"?

**Reply by tummerry** [reply]
app YI Home and YI Life

**Reply by roleoroleo** [reply]
Yes, if the cam is for the chinese market, you have to use proxychains to use the app.
Some users changed the serial number from CH to US and the cam started to speak in english but I think the app doesn't work with this trick.

**Reply by tummerry** [reply]
ok

> Yes, if the cam is for the chinese market, you have to use proxychains to use the app. Some users changed the serial number from CH to US and the cam started to speak in english but I think the app doesn't work with this trick.

ok
